pkg/kube/translation/annotations/types.go

```diff
@@ -18,6 +18,10 @@ import (
 	"strings"
 )
 
+const (
+	AnnotationsPrefix = "k8s.apisix.apache.org/"
```

Comment:
Is this constant exported to the outside? If not, we can rename it to `annotationsPrefix`.

Reply:
this will be used in `translation` package to dicide whether use-regex annotation is true

Reply:
OK, let's add some comments for exportable constants.

Reply:
done